### DISCUSSION TITLE
githubstatus.com: Unsubscribing from status updates

### DISCUSSION THREAD

**Discussion by orpheuslummis** [community/community#12981] [Other Feature Feedback, Questions, & Ideas] [answered]
I subscribed to status updates (email and SMS) on https://www.githubstatus.com/. I would like to unsubscribe but didn't find the capability to do so.

**Comment by orpheuslummis** [comment, marked as answer]
I found the way to unsubscribe: it is via a link sent in the initial subscription confirmation.
I would recommend making it a bit more prominent!

**Reply by lindacrosby603** [reply]
I definitely agree that unsubscribe should be found much easier...my personal opinion.

**Comment by basiphobe** [comment]
I couldn't find the initial subscription confirmation, but I did scroll all the way back through my text/sms message history, and found the first text sent:

![image](https://user-images.githubusercontent.com/1565483/193893611-1afb4f44-bc2c-4bfc-b8ec-34e31db5dc47.png)

Replying STOP worked to get me unsubscribed.

![image](https://user-images.githubusercontent.com/1565483/193893722-d1fb5962-2e77-44cb-9a05-d3f53fc53bfb.png)

**Comment by crunk1** [comment]
I've tried unsubscribing via the unsubscribe link in the emails, e.g. [https://www.githubstatus.com/?unsubscribe=mysubscriptioncode](https://www.githubstatus.com/?unsubscribe=mysubscriptioncode), but clicking the "UNSUBSCRIBE FROM UPDATES" button is not working? I tried reverse engineering the unsubscribe page to POST the form myself, but to no success.
